fe/fe-core/src/main/java/com/starrocks/catalog/ListPartitionInfo.java

```diff
@@ -599,7 +592,7 @@ public static int compareByValue(List<List<String>> left, List<List<String>> rig
     }
 
     public void setStorageCacheInfo(long partitionId, DataCacheInfo dataCacheInfo) {
```

Comment:
better to remove this function and use `setDataCacheInfo` directly

Reply:
done